src/commands/Generation.ts

```diff
@@ -0,0 +1,53 @@
+import { Client, Message, TextChannel } from "discord.js";
+import { ComplimentGenerator, InsultGenerator, JokeGenerator } from "../generators";
+import { CommandType, ICommand } from "./Command";
+
+/*
+  Extension of the base Command interface for the Generate command.
+*/
+export interface IGenerationCommand extends ICommand {
+  // This is the channel ID to explicitly send the message to.
+  channelIDToSendMessage: string;
+  message: string;
```

Comment:
Rename `message` to `generationType`. Maybe have an enum for the Generation types.

Reply:
done